test/benchmarks/utility_scale.py

```diff
@@ -48,7 +48,7 @@ def setup(self, basis_gate):
         self.qaoa_qasm = os.path.join(qasm_dir, "qaoa_barabasi_albert_N100_3reps.qasm")
         self.qaoa_qc = QuantumCircuit.from_qasm_file(self.qaoa_qasm)
         self.qv_qc = build_qv_model_circuit(50, 50, SEED)
-        self.circSU2 = EfficientSU2(100, reps=3, entanglement="circular")
+        self.circSU2 = efficient_su2(100, reps=3, entanglement="circular")
```

Comment:
This will change the benchmarked behavior as the circuit now comes unwrapped, but I'd say this is fine since we've changed the workflow here. If we wanted to benchmark decomposition, we could add another test specifically for that in the future.